### ISSUE TITLE
feat: Hook upon upload/download

### ISSUE THREAD

**Issue by drehelis** [coderaiser/cloudcmd#390]
Is it possible to have an option to fire a POST/GET request to an external endpoint after a successful operation?

**Comment by coderaiser** [comment]
Could you please provide more details?

**Comment by drehelis** [comment]
What I'm looking for is an option to be able to upload a file using `cloudcmd` and upon successful upload operation, to be able to notify an external service to reload/re-read that data dir.

**Comment by coderaiser** [comment]
You can use `PUT` method of [restafary](https://github.com/coderaiser/node-restafary#rest) for this purpose.

**Comment by coderaiser** [comment]
You can implement this with User Menu,ask if you have any questions on this.

**Comment by drehelis** [comment]
@coderaiser Where can I see an example?

**Comment by coderaiser** [comment]
https://cloudcmd.io/#user-menu

**Comment by drehelis** [comment]
Using `User Menu`, means user needs to hit certain key to execute the desired operation?
What If I want it automatically for each file I upload or delete?

**Comment by coderaiser** [comment]
Yes it would be better if user download file with User Menu, in this case you will have much more control over the process.

**Comment by drehelis** [comment]
My setup of cloudcmd has the lower menu hidden, User Menu is not used.
Is it possible to do this via upload context menu?
![image](https://github.com/coderaiser/cloudcmd/assets/11921235/9d5522bb-53f6-4e81-917a-49ffbe75aad8)

**Comment by coderaiser** [comment]
That's not sounds like a feature that everybody needs, and going to use. Could you show me how API should look like, so you can use it?
